apps/meteor/client/sidebarv2/RoomList/RoomList.tsx

```diff
@@ -49,7 +49,7 @@ const RoomList = () => {
 
 	return (
 		<Box position='relative' display='flex' overflow='hidden' height='full' flexGrow={1} flexShrink={1} flexBasis='auto' ref={ref}>
-			<VirtualizedScrollbars>
+			<VirtualizedScrollbars width='full'>
```

Comment:
instead of adding this prop, we should check if its parent needs to have all these attributes, mainly `display: flex`